ucsb_ds_capstone_projects_2021/projects/calcofi1/update2.md

```diff
@@ -77,9 +77,9 @@ fish that is to be caught in the future years because more fish are becoming adu
 from those larvae. 
 
 ### d) Total Catch vs. Larvae Amounts (with years)
-<img src=https://raw.githubusercontent.com/gavintieng/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/calcofi1/meanlarvaevscatchlbsopah.png alt="drawing" />
+<img src=https://raw.githubusercontent.com/gavintieng/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/calcofi1/opah_fixedc.png alt="drawing" />
 
-<img src=https://raw.githubusercontent.com/gavintieng/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/calcofi1/meanlarvaevscatchlbssardine.png alt="drawing">
+<img src=https://raw.githubusercontent.com/gavintieng/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/calcofi1/sardine_fixedc.png alt="drawing">
```

Comment:
Same comment as above

Markdown can embed images directly: https://jupyterbook.org/content/figures.html
You want to add the image `update2_exploratory.png` to the same directory and replace the HTML with `![some-text](update2_exploratory.png)`